File: Pascal/Task240.pas
program Task240;
const eps = 0.00001;
var x, s, u, f : double;
n: integer;
begin
    writeln('Введіть значення змінної x');
    readln(x);
    s := 1; n := 2; u := x / 4;
    while abs(u) > eps do 
    begin
        s := s + u;
        u := -x * (4 * n - 5) / (4 * n) * u;
        n := n + 1
    end;
    f :=  sqrt(sqrt(1 + x));
    {корінь 4-степеня можна записати і так}
    writeln('Сума ряду s = ', s:12:5);
    writeln('Значення функції f = ', f:12:5);
    writeln('Кількість доданків : ', n - 1 )
end.

File: Kotlin/Task240.kt
import java.lang.Math.pow
import kotlin.math.abs
import kotlin.math.pow

fun main() {
    println("Введіть значення x")
    val x: Double = readln().toDouble()
    var u: Double = x / 4
    var s = 1.0
    val eps = 0.00001
    var n = 2
    while (abs(u) > eps) {
        s += u
        u *= -x * (4 * n - 5) / (4 * n)
        n++
    }
    println("Значення суми s = %.6f".format(s))
    println("Значення функції: %.6f"
        .format((1 + x).pow(0.25)))
    println("Кількість доданків " + (n - 1))
}
